bin/mock_onfido.pl

```diff
@@ -311,6 +351,25 @@ sub create_report {
     return $c->render(json => $sdk_token);
 };
 
+post '/backdoor' => sub {
```

Comment:
what does `backdoor` specify here?

Reply:
I think test will be easier if we have a backdoor (or backhole ?)。 So we can control the data of mocked server easily. For example, when we want the mock_server change the report from 'in_progress' to 'complete' , we can post to `backdoor` to update the report status directly.

I didn't use it yet . But I think it is a useful feature.  Maybe I should delete it now and add it back when I really need it .

Reply:
Removed.